xrpl4j-crypto-parent/xrpl4j-crypto-core/src/main/java/org/xrpl/xrpl4j/crypto/core/signing/SignatureSerializer.java

```diff
@@ -0,0 +1,17 @@
+package org.xrpl.xrpl4j.crypto.core.signing;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.JsonSerializer;
+import com.fasterxml.jackson.databind.SerializerProvider;
+
+import java.io.IOException;
+
+/**
+ * Custom Jackson serializer for {@link Signature}es.
+ */
+class SignatureSerializer extends JsonSerializer<Signature> {
```

Comment:
Let's move this and the deserializer into the same package as the other serializers/deserializers and add them to the `Xrpl4jModule` so you don't need the `using = SignatureSerializer` and `using = SignatureDeserializer.class`

Reply:
This still needs to be addressed, correct?

Reply:
The problem is that `Signature` is in xrpl4j-crypto, which depends on `xrpl4j-model`, but all of the Jackson modules are in `xrpl4j-model`, so there's no way to move these without creating a circular dependency. We probably want to extract all the Jackson stuff into it's own module at some point, but I think that should be future work. WDYT?